lib/checkunusedvar.cpp

```diff
@@ -1534,7 +1534,14 @@ void CheckUnusedVar::checkStructMemberUsage()
         if (bailout)
             continue;
 
+        // Keep track of number of non-static members encountered so far
+        int nummembs = 0;
```

Comment:
naming..  I like `someName` use a capital letter for first letter in each word. And I prefer full name. "membs" is much harder to read than "members" and you only save 2 characters.